sandbox/rocky/tf/plotter/plotter.py

```diff
@@ -0,0 +1,90 @@
+import atexit
+from multiprocessing import Process
+import numpy as np
+import pickle
+import platform
+from queue import Empty, Queue
+import tensorflow as tf
+from threading import Thread
+
+from rllab.sampler.utils import rollout
+
+__all__ = ['init_worker', 'init_plot', 'update_plot']
+
+process = None
+queue = None
+sess = None
```

Comment:
If you pass sess as a parameter to _worker_start using the Thread constructor, there's no need to have this as a global variable.